EasyClangComplete.py

```diff
@@ -174,12 +176,14 @@ def on_post_save_async(self, view):
 
         """
         # disable on_activated_async when running tests
+        self.saving = True
```

Comment:
Why do we need this here? At least it should be moved a couple lines lower, as the comment above it doesn't make sense anymore. Anyway, I guess in this way you are trying to guarantee that the query of the hint does not happen simultaneously with saving. But this should probably be done with a mutex on shared resources as opposed to a boolean variable. Correct me if I misunderstood the intention.